### PR TITLE
CESQL v1 release notes

### PR DESCRIPTION
**Pull request by Cali0707**
## Proposed Changes

- Add release notes for CESQL v1
- Link to release notes in RELEASES.md
- Align RELEASES.md table

### CODE REVIEW

**Comment by Cali0707** [comment]
cc @duglin

File: docs/RELEASES.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-| CloudEvents         | [1.0](https://github.com/cloudevents/spec/tree/v1.0) | 2019/10/24 | [Notes](../cloudevents/RELEASE_NOTES.md#v100---20191024) |
-| CloudEvents         | [0.3](https://github.com/cloudevents/spec/tree/v0.3) | 2019/06/13 | [Notes](../cloudevents/RELEASE_NOTES.md#v03---20190613) |
-| CloudEvents         | [0.2](https://github.com/cloudevents/spec/tree/v0.2) | 2018/12/06 | [Notes](../cloudevents/RELEASE_NOTES.md#v02---20181206) |
-| CloudEvents         | [0.1](https://github.com/cloudevents/spec/tree/v0.1) | 2018/04/20 | [Notes](../cloudevents/RELEASE_NOTES.md#v01---20180420) |
+| Specification Group | Version                                                              | Release Date | Release Notes                                            |
+| :------------------ | :------------------------------------------------------------------: | :----------- | :------------------------------------------------------: |
+| CESQL               | 1.0                                                                  | 2024/06/13   | [Notes](../cesql/RELEASE_NOTES.md#v100---20240613)       |
```
Comment (duglin) [inline docs/RELEASES.md:5]: let's make `1.0` a link to: https://github.com/cloudevents/spec/tree/cesql/v1.0.0
the link checker will fail until we create the new release/branch but that's ok

**Comment by duglin** [comment]
I forgot one more file: https://github.com/cloudevents/spec/blob/main/README.md  update the table to reference `v1.0.0` with a link to: https://github.com/cloudevents/spec/tree/cesql/v1.0.0/cesql
We should also update the README.md in the cesql to have more than just a link to the spec - just a short paragraph (even stolen from the spec) will be good enough for now

**Comment by Cali0707** [comment]
Sorry for the delay @duglin - I think everything should be correct now

**Comment by duglin** [comment]
Merging!

**Pull request merged**